avm/utilities/pipelines/e2eValidation/resourceDeployment/Test-TemplateDeployment.ps1

```diff
@@ -116,6 +116,12 @@ function Test-TemplateDeployment {
 
         # Additional parameter object provided yes/no
         if ($AdditionalParameters) {
+            # Convert any secure strings to plain text
+            # foreach ($name in @($AdditionalParameters.Keys)) {
+            #     if ($AdditionalParameters[$name] -is [System.Security.SecureString]) {
+            #         $AdditionalParameters[$name] = ConvertFrom-SecureString $AdditionalParameters[$name] -AsPlainText
+            #     }
+            # }
```

Comment:
??????

Reply:
Turns out, a Bicep template parameter that has the `@secure()` annotation only accepts SecureStrings - which is great, because that meant I did not need to implement the conversion from a key vault secure string secret to a plain text string afterall. Removing the leftover...